DEVELOPMENT.md

```diff
@@ -101,41 +101,41 @@
 
 ## Test Running
 
--   Run all unit tests.
+-   Run all unit and integration tests.
 
     ```sh
-    bazel test //...
+    bazel test ...
     ```
 
--   Run a single unit test like `JavaCodeGeneratorTest.java`
+-   Run all unit tests.
 
     ```sh
-    bazel run //src/test/java/com/google/api/generator/engine:JavaCodeGeneratorTest
+    bazel test $(bazel query ... | grep ^//:unit_)
     ```
 
--   Run a single integration test for API like `Redis`, it generates Java source
-    code using the Java microgenerator and compares them with the goldens files
-    in `test/integration/goldens/redis`.
+-   Run a single unit test like `JavaCodeGeneratorTest.java`
 
     ```sh
-    bazel test //test/integration:redis
+    bazel run :unit_com_google_api_generator_engine_JavaCodeGeneratorTest
     ```
 
--   Run all unit and integration tests.
+-   Update unit test golden files, for example `JavaCodeGeneratorTest.java`:
 
     ```sh
-    bazel test $(bazel query "src/test/..." | grep "Test$") //test/integration/...
+    bazel run :update_com_google_api_generator_engine_JavaCodeGeneratorTest
     ```
 
--   Update unit test golden files, for example `JavaCodeGeneratorTest.java`:
+-   Run a single integration test for API like `Redis`, it generates Java source
+    code using the Java microgenerator and compares them with the goldens files
+    in `test/integration/goldens/redis`.
 
     ```sh
-    bazel run //src/test/java/com/google/api/generator/engine:JavaCodeGeneratorTest_update
+    bazel test test/integration:redis
     ```
 
 -   Update integration test golden files, for example `Redis`. This clobbers all the
     files in `test/integration/goldens/redis`.
 
     ```sh
-    bazel run //test/integration:redis_update
+    bazel run test/integration:update_redis
```

Comment:
Is it possible to update all goldens in one command?

Reply:
Like we document that one should do `bazel test $(bazel query ... | grep ^//:unit_)` to run unit tests, one can do

`bazel test $(bazel query test/integration/... | grep update_)`

I'll document this.